packages/mosaic/core/src/QueryManager.ts

```diff
@@ -105,6 +114,52 @@ export class QueryManager {
 
       if (cache) this.clientCache!.set(sql!, data);
 
+      // Emit 'queries:end' event
```

Comment:
Let's not add comments than explain code. Add comments that explain why, not what.